docs/src/examples/components/Toolbar/Content/ToolbarExampleCustomContent.shorthand.tsx

```diff
@@ -3,6 +3,7 @@ import { Button, Text, Toolbar } from '@stardust-ui/react'
 
 const ToolbarExampleCustomContentShorthand = () => (
   <Toolbar
+    aria-labelledby="toolbar-can-contain-custom-content"
```

Comment:
use `aria-label` as the examples should not reference anything outside of it

Reply:
done :)